docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm

```diff
@@ -13,23 +13,29 @@ <h1>Checkbox and Radio Button</h1>
 
 <p>Checkbox and Radio Button components shows selectable items:</p>
 
+<h2>Checkbox</h1>
 <img src="../../images/checkbox-wearable.png" alt="photo of checkboxRadio in Tizen">
```

Comment:
"checkboxRadio" - I think this is bug. I think should be "photo of Checkbox and Radio button in Tizen"

Reply:
fixed